code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -66,9 +66,26 @@
 	can_hold = list() // any
 	cant_hold = list("/obj/item/weapon/disk/nuclear")
 
-	suicide_act(mob/user)
-		viewers(user) << "<span class='danger'>[user] puts the [src.name] over \his head and tightens the handles around \his neck! It looks like \he's trying to commit suicide.</span>"
-		return(OXYLOSS)
+	slot_flags = SLOT_BELT | SLOT_HEAD
+	flags = FPRINT | BLOCK_BREATHING

```

Comment:
BLOCK_GAS_SMOKE_EFFECT ?
I'm assuming if you couldn't breath that you wouldn't be able to inhale smoke.